tabbycat/tournaments/templates/round_advance_check.html

```diff
@@ -1,26 +1,25 @@
 {% extends "base.html" %}
-{% load debate_tags %}
+{% load i18n debate_tags %}
 
-{% block page-title %}Confirm Round Advance{% endblock %}
+{% block page-title %}{% trans "Confirm Round Advance" as page_title %}{% endblock %}
 {% block head-title %}
-  <span class="emoji">🙏</span>Confirm Round Advance
+  <span class="emoji">🙏</span>{{ page_title }}
```

Comment:
Just to make things even more confusing, this doesn't work either. Variables defined outside a `block` can't be used inside it: https://docs.djangoproject.com/en/2.0/ref/templates/language/#template-inheritance

It might seem like it violates DRY but I think it's actually fine to have `{% trans "Confirm Round Advance" %}` in there twice, as you might notice we often do. It's just UI text, and the `gettext` tool picks up on duplicate strings and lists them in the translation files only once, unless you provide a `context` to differentiate between them.